plugin/verilog_systemverilog.vim

```diff
@@ -169,6 +169,12 @@ let g:verilog_syntax = {
                         \ 'highlight'   : 'verilogStatement',
                         \ 'syn_argument': 'transparent keepend contains=ALLBUT,verilogInterface',
                         \ }],
+      \ 'ovm'         : [{
+                        \ 'match_start' : '`\<\(ovm\|uvm\)_\a\+_utils_begin\>',
```

Comment:
`[ou]vm` ? ;)

Reply:
In hindsight that does seem a little simpler ;)